src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -84,29 +88,36 @@
          ;; sheet
          [gesture/gesture-detector {:gesture sheet-gesture}
           [reanimated/view
-           {:style     (reanimated/apply-animations-to-style
-                        {:transform [{:translateY translate-y}]}
-                        (style/sheet insets
-                                     window-height
-                                     theme
-                                     padding-bottom-override
-                                     selected-item
-                                     shell?))
-            :on-layout #(reset! sheet-height (oops/oget % "nativeEvent" "layout" "height"))}
+           {:style (reanimated/apply-animations-to-style
+                    {:transform [{:translateY translate-y}]}
+                    (style/sheet insets
+                                 window-height
+                                 theme
+                                 padding-bottom-override
+                                 selected-item
+                                 shell?))}
            (when gradient-cover?
              [rn/view {:style style/gradient-bg}
               [quo/gradient-cover {:customization-color customization-color}]])
            (when shell?
              [blur/ios-view {:style style/shell-bg}])
            (when selected-item
              [rn/view
-              [rn/view {:style (style/selected-item theme window-height @sheet-height insets)}
-               [selected-item]]])
+              {:on-layout (fn [event]
+                            (reset! item-height (oops/oget event "nativeEvent" "layout" "height")))
+               :style     (style/selected-item theme top @sheet-height show-bottom-margin border-radius)}
+              [selected-item]])
 
-           ;; handle
-           [rn/view {:style (style/handle theme)}]
-           ;; content
-           [content]]]]))))
+           [rn/view
+            {:style     (when selected-item
+                          (style/sheet-content theme padding-bottom-override insets))
+             :on-layout (fn [event]
+                          (reset! sheet-height
+                            (oops/oget event "nativeEvent" "layout" "height")))}
+            ;; handle
+            [rn/view {:style (style/handle theme)}]
+            ;; content
```

Comment:
Since you are here, and if you want to do some clean-up, these comments `handle`/`content` are completely unnecessary.